src/test/kotlin/StringCalculatorTest.kt

```diff
@@ -0,0 +1,62 @@
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.assertj.core.api.AssertionsForClassTypes.assertThat
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import org.junit.platform.commons.util.StringUtils
+import java.lang.IllegalArgumentException
+
+class StringCalculatorTest {
+    /**
+     * 공백 검사 제대로 작동?
+     * 사칙연산 외 기호 예외처리 작동?
+     * 올바른 값 변환?
+     */
+
+
+    @Test
+    fun checkCalculableTest() {
+        assertThatThrownBy { checkCalculable(listOf("&")) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("사칙연산 기호 쓰셔야 합니다!!!")
+    }
+
+    @Test
+    fun calculateTest() {
+        assertThat(calculate(listOf("1", "+", "7", "-", "2", "*", "3", "/", "2")))
+            .isEqualTo(9)
+    }
+
+    @Test
+    fun changeToIntTest() {
+        assertThat(1).isEqualTo(changeToInt("1"))
+    }
+
+    @Test
+    fun sumDataTest() {
+        assertThat(1 + 2).isEqualTo(sumData(1, 2))
+    }
```

Comment:
```suggestion
    @Test
    fun sumDataTest() {
        assertThat(sumData(1, 2)).isEqualTo(3)
    }
```
이런식으로 순서를 바꾸는 게 좀 더 가독성이 좋아보여!
that 이하를 assert 하겠다 라는 뜻이니까 실제 대상보다 비교 대상이 먼저 오는게 맞아보여!